fix(APM): Removed Explorer option from the all capabilities section (#13828)
newrelic/docs-website: src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

## How to find the external services feature [#where-externals]

> The external services feature is available from the left navigation pane once you select a service in Explorer. You can open it by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & Services**. Then, click your app and, under the **Monitor** section, click **External services**.

Reviewer: ```suggestion
To access the external services feature, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & Services**. Select an app, then under the **Monitor** section click **External services**.
```

Reviewer: @nbaenam , I was told we bold the entire UI path, including the `>`.

Author: Alba told me the opposite, that's why I changed it... I'm going to change it again. Thank you for letting me know.
